package.json

```diff
@@ -69,7 +69,7 @@
     "build": "react-scripts build",
     "lint": "eslint --ext js,ts,tsx src",
     "test": "react-scripts test --env=jest-environment-jsdom-sixteen",
-    "test:changed": "npm run test -- -o --watchAll=false",
+    "test:changed": "npm test -- -o --watchAll=false",
```

Comment:
Maybe `yarn test -- -o --watchAll=false` to be consistent

Reply:
updated